### PR TITLE
added Instagram integration

### PR DESCRIPTION
**Pull request by advaithm**
I made this quick and simple integration for Instagram.  it should be as simple as copy pasting this with some minor changes to link to another social media. This section needs to be added to the config.toml
```toml
[params.social.instagram]
    name= "instagram"
    url="link to your account"
    icon="fab fa-instagram fa-2x"
``` 
There should be a better way of linking this. like iterating over a bunch of social media configs and linking them with this html.
```html
<a href="{{ $social.url}}" name="$social.name"><i
            class="{{$social.icon}}"></i></a>
```
 but I couldn't really figure out the stuff i needed to do on the config.toml end of things.
This is in reference to issue #27

### CODE REVIEW

File: layouts/partials/social.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div>
     {{ if .Site.Params.social.instagram }}
-    <a href="{{.Site.Params.social.url}}" name=".Site.Params.social.instagram.name"><i
+    <a href="{{.Site.Params.social.instagram.url}}" name=".Site.Params.social.instagram.name"><i
```
Comment (matsuyoshi30) [inline layouts/partials/social.html:3]: I think the name attribution should use `{{}}` like the href.

**Comment by matsuyoshi30** [comment]
@advaithm Thank you for PR! I reviewed and commented, so please check it.

**Comment by advaithm** [comment]
I will make the changes that you have recommended.

**Comment by advaithm** [comment]
@matsuyoshi30 I have made the changes you recommend please look into this

**Comment by matsuyoshi30** [comment]
@advaithm LGTM. Thanks!

**Pull request merged**